eng/mgmt/mgmtmetadata/compute_resource-manager.txt

```diff
@@ -12,4 +12,4 @@ Branch:      master
 Commit:      10873ac628ee703126f82974487d8a290e983f8c
```

Comment:
Hi @grizzlytheodore 
I think commit ID `10873ac628ee703126f82974487d8a290e983f8c` points to the `2021-03-01` version of compute Swagger API: https://github.com/Azure/azure-rest-api-specs/blob/10873ac628ee703126f82974487d8a290e983f8c/specification/compute/resource-manager/readme.md

You need to update it to the commit ID of `2021-04-01`.

Reply:
@archerzz thank you for pointing this out. I've updated to the commit ID.